lib/network_manager.go

```diff
@@ -190,8 +190,10 @@ func (nm *NetworkManager) startValidatorConnector() {
 			nm.exitGroup.Done()
 			return
 		case <-time.After(nm.peerConnectionRefreshInterval):
+			glog.V(2).Infof("NetworkManager.startValidatorConnector: Refreshing validator indices...")
```

Comment:
Sorry I should have been more clear. Let's also delete these glogs because they're too verbose. They just print every second.